tests/PySys/local_mqtt/local_mqtt_pub_sub_retain/run.py

```diff
@@ -0,0 +1,65 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate local publishing with retain flag and subscribing:
+
+Given a configured system
+When we start tedge pub with sudo to publish a message with retain flag to temperature sensor alaram topic
+When we start tedge pub with sudo to publish an empty message with retain flag to pressure sensor alarm topic
+When we start tedge sub with sudo in the background to subscribe to tempearture sensor alarm topic
+hen we start tedge sub with sudo in the background to subscribe to pressure sensor alarm topic
+Then we kill tedge sub with sudo as it is running with a different user account
+Then we find the messages in the output of temperature and pressure sensor subscribers logs
```

Comment:
Maybe "Then we find the retained messages..." that would make it a bit clearer for the reader

Reply:
Updated